Peekabook/Peekabook/Presentation/Recommend/VC/ConfirmPopUpViewController.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  ConfirmPopUpViewController.swift
+//  Peekabook
+//
+//  Created by 고두영 on 2023/01/04.
+//
+
+import UIKit
+
+import SnapKit
+import Then
+
+import Moya
+
+final class ConfirmPopUpViewController: UIViewController {
+    
+    // MARK: - Properties
+
+    // MARK: - UI Components
+    private let popUpView = UIView().then {
+        $0.backgroundColor = .peekaBeige
+    }
+    
+    private var confirmLabel = UILabel().then {
+        $0.numberOfLines = 0
+        $0.textAlignment = .center
+        $0.text = "고두영님에게\n책을 추천하시겠어요?"
+        $0.font = .h4
+        $0.textColor = .peekaRed
+    }
+    
+    private lazy var cancelButton = UIButton().then {
+        $0.addTarget(self, action: #selector(touchCancelButton), for: .touchUpInside)
```

Comment:
p3
버튼을 눌렀을 때 동작하는 함수니까 명명을 이렇게 바꿔도 좋아보여요~
```suggestion
        $0.addTarget(self, action: #selector(touchCancelButtonDidTap), for: .touchUpInside)
```